libs9s/s9soptions.cpp

```diff
@@ -4498,6 +4499,17 @@ S9sOptions::isListDb3dVersionsRequested() const
     return getBool("list_3d_versions");
 }
 
+/**
+ * \returns true if the "use_vendor_api" function is requested by providing the --use-vendor-api
+ *   command line option.
+ */
+bool
+S9sOptions::isUseVendorApiRequested() const
+{
+    return getBool("use_vendor_api");
```

Comment:
`This option must be used in combination with --list-3d.`
Taking this into account should `true` be returned only if `isListDb3dVersionsRequested()` also returns true?